### PR TITLE
Update IsogramTestGenerator.scala. Update IsogramTest.scala. Refs #37…

### PR DESCRIPTION
**Pull request by ricemery**
* Update IsogramTestGenerator.scala. 
* Update IsogramTest.scala. 
* Replace empty impl file with `.keep`

Refs #370, #331

### CODE REVIEW

File: exercises/isogram/src/test/scala/IsogramTest.scala [hunk 1]
```diff
@@ -1,52 +1,49 @@
...
+import org.scalatest.{Matchers, FunSuite}
 
+/** @version 1.1.0 */
 class IsogramTest extends FunSuite with Matchers {
-
+  
   test("empty string") {
-    val result = Isogram.isIsogram("")
-    result should be (true)
+    Isogram.isIsogram("") should be (true)
```
Comment (ErikSchierboom) [inline exercises/isogram/src/test/scala/IsogramTest.scala]: Are the parentheses in `(true)` necessary?

**Comment by ricemery** [comment]
The parens are necessary if "should be" is used. If "shouldBe" is used as
the matcher then no parens are needed.

I am not sure that one style is preferred over the other. I have just
always used "should be".

**Comment by ErikSchierboom** [comment]
Ah okay. I think it's fine then!

**Comment by ricemery** [comment]
Thanks!

**Pull request merged**